parquet-column/src/main/java/org/apache/parquet/internal/column/columnindex/OffsetIndex.java

```diff
@@ -64,4 +66,11 @@ public default long getLastRowIndex(int pageIndex, long rowGroupRowCount) {
     int nextPageIndex = pageIndex + 1;
     return (nextPageIndex >= getPageCount() ? rowGroupRowCount : getFirstRowIndex(nextPageIndex)) - 1;
   }
+
+  /**
+   * @param pageIndex
+   *          the index of the page
+   * @return unencoded/uncompressed size for BYTE_ARRAY types; or empty for other types.
```

Comment:
It might be worth clarifying that even for BYTE_ARRAY types this value might not have been written?